app/templates/components/event-table.hbs

```diff
@@ -0,0 +1,58 @@
+<div class = "ui grid container">
+  <div class = "row">
+    <div class = "sixteen wide column">
+      <table class="ui compact  celled table">
+        <thead>
+          <tr>
+            <th>Name</th>
+            <th>Date</th>
+            <th>Roles</th>
+            <th>Sessions</th>
+            <th>Speakers</th>
+            <th>Tickets</th>
+            <th>Public Url</th>
+            <th>Actions</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td>
+              <div><i class="large camera icon"></i></div>
+              <div>Abhinav Khare</div>
+            </td>
+            <td>
+              27-06-2017
+            </td>
+            <td>
+              <ol>
+                <li>abhinav.khare31@gmail.com (Organizer)</li>
+              </ol>
+            </td>
+            <td>
+              <ol>
+                <li>Drafts: 0</li>
+                <li>Submitted: 0</li>
+                <li>Accepted: 0</li>
+                <li>Confirmed: 0</li>
+                <li>Pending: 0</li>
+                <li>Rejected: 0</li>
+              </ol>
+            </td>
+            <td>
+              2
+            </td>
+            <td>
+              No Ticket information
+            </td>
+            <td>
+              localhost/event
+            </td>
+            <td>
+              <i class = "large delete icon"></i>
```

Comment:
@niranjan94 Is it a good idea to use colours for significance eg : 
![image](https://cloud.githubusercontent.com/assets/17252805/25989640/9202dd8a-371a-11e7-840d-e14d4d7a1043.png)